src/Core/src/Handlers/View/ViewHandler.cs

```diff
@@ -58,7 +58,10 @@ public abstract partial class ViewHandler : ElementHandler, IViewHandler
 #if WINDOWS || MACCATALYST
 				[nameof(IContextFlyoutElement.ContextFlyout)] = MapContextFlyout,
 #endif
-			};
+#if ANDROID
+				["Initialize"] = (h, v) => ((PlatformView?)h.PlatformView)?.Initialize(v)
```

Comment:
@hartez @jonathanpeppers any reason we can't move the Initialize code to the mapper? 

The problem with having this call outside the mapper is that classes implementing `IViewHandler` but not our `ViewHandler` will now fail to initialize any properties that we are batching. 

Example: https://github.com/dotnet/maui/pull/10672/files#r997579414

It seems like all the tests are passing if we move the `initialize` here.

Reply:
I remember there was a general architectural problem, where I couldn't apply this treatment to more controls: `Label`, `Image`, etc. were some ideas. I believe any property that is remapped to `Initialize`, you can no longer override.

If you make this change, you might test these benchmarks before/after:

https://github.com/dotnet/maui/blob/f1bd01c9378ca5511e50a2eb4cff803845fd1594/src/Core/tests/Benchmarks.Droid/ViewHandlerBenchmark.cs#L52-L74

I wouldn't change this in .NET 7 (only 8 at this point).

Reply:
But does `Opacity` still get mapped? The idea was we'd avoid `Opacity` and the other ~20 properties and `Initialize` would set all of those values.